pkg/controller/controller_test.go

```diff
@@ -61,15 +61,15 @@ import (
 // loops for the different catalog API resources.
 
 const (
-	testClusterServiceClassGUID            = "CSCGUID"
-	testClusterServicePlanGUID             = "CSPGUID"
-	testNonbindableClusterServiceClassGUID = "UNBINDABLE-CLUSTERSERVICECLASS"
-	testNonbindableClusterServicePlanGUID  = "UNBINDABLE-CLUSTERSERVICEPLAN"
-	testServiceInstanceGUID                = "IGUID"
-	testServiceBindingGUID                 = "BGUID"
+	testClusterServiceClassGUID            = "cscguid"
+	testClusterServicePlanGUID             = "cspguid"
```

Comment:
lot of places that these constants can be used instead of changing the hardcoded strings.